.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!-- Instructions For Filing a Bug: https://github.com/pydicom/imbalanced-learn/blob/master/CONTRIBUTING.md#filing-bugs -->
+
+#### Description
+<!-- Example: xxxxxx Error thrown when calling xxx on xxxxxxxxxxxxxxxxxxxxxxxxx with xxxxxxxxxxxxxx > 0-->
+
+#### Steps/Code to Reproduce
+<!--
+Example:
+```py
+from xxxxxxx.xxxxxxxxxxxxxxxxxx.xxxx import xxxxxxxxxxxxxxx
+from xxxxx.xxxxxxxxxxxxxxxxxx import xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+docs = ["Help I have a bug" for i in range(1000)]
+
+xxxxxxxxxx = XxxxxXxxxxxxxxx(xxxxx=xxxx, xxxxxxxx=xxxxxx)
+xxx_xxxxxxxx = xxxxxxxxxx.xxx_xxxxxxxxx(xxxx)
+
+xxx_xxxxx = XxxxxxXxxxxxxxxXxxxxxxxxx(
+    x_xxxxxx=xx,
+    xxxxxxxx_xxxxxx=xxxxxxxx,
+    xxxxxxxx_xxxxx=xx,
+    x_xxxx=x,
+)
+xxxxx = xxx_xxxxx.xxx(xxx_xxxxxxxx)
+```
+If the code is too long, feel free to put it in a public gist and link
+it in the issue: https://gist.github.com
+-->
+
```

Comment:
The user should also include an anonymised version of the dataset they're having issues with (if possible).